File: Findthedifference.c
#include <stdio.h>
#include <string.h>

char findTheDifference(char* s, char* t) {
    char ans=0;
    for(int i = 0 ; s[i]!='\0'; i ++)ans^=(s[i]^t[i]);
    ans^=t[strlen(t)-1];
    return ans;
}

File: Findthedifference.py
#!/usr/bin/env python
# coding=utf-8
class Solution:
    def findTheDifference(self, s, t):
        """
        :type s: str
        :type t: str
        :rtype: str
        """
        ans = 0
        for elem in s + t:
            ans ^= ord(elem)
        return chr(ans)
